health_check/contrib/twilio/apps.py

```diff
@@ -0,0 +1,20 @@
+from django.apps import AppConfig
+from django.conf import settings
+
+from health_check.plugins import plugin_dir
+
+
+class HealthCheckConfig(AppConfig):
+    name = 'health_check.contrib.twilio'
+
+    def ready(self):
+        from .backends import TwilioHealthCheck
+
+        if getattr(settings, 'HEALTHCHECK_FANOUT_BACKENDS', True):
+            for service in getattr(settings, 'HEALTHCHECK_TWILIO_SERVICES', ['SMS', 'Phone Numbers']):
+                celery_class_name = '{}:{}'.format(TwilioHealthCheck.__name__, service.replace(' ', ''))
+
+                celery_class = type(celery_class_name, (TwilioHealthCheck,), {'queues': [service]})
+                plugin_dir.register(celery_class)
```

Comment:
hmm... it might be that just don't understand completely what's going on here but this whole block looks a bit like a bad copy paste to me. If I understand you're intentions correctly your're registering different plugins for different Twilio Services, e.g. one for "SMS", one for "Phone Numbers", correct? You should change the naming then to not use `celery` because this has nothing to do with celery if I get it correctly. Also: shouldn't it be `{'service': [service]})` instead of `{'queues': [service]})`? Because you're using `self.services = getattr(self, 'services', [])` later in the base class.
Personally, I agree with @codingjoe, I would not set a default here for the services you want to check. That's not intuitive. Also it feels like one setting is enough to configure the health check for different services instead of `HEALTHCHECK_FANOUT_BACKENDS` and `HEALTHCHECK_TWILIO_SERVICES`. The latter one should be enough. If the setting is not configured, just register the normal `TwilioHealthCheck`.

Reply:
Yup, copy/paste error. Sorry about that. Like I said, I was trying to cobble this PR together from my own branch, which more uniquely matches my use case. If I get some time, perhaps I will try to broaden out/add more testing for these PRs to make them more universally applicable if no one else gets to them first.